### PR TITLE
[Enhancement] hide hudi internal fields

### PR DESCRIPTION
**Pull request by dirtysalt**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Some hudi internal fields(startswith _hoodie) seem useless, maybe we can hide them out when building schema.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java [hunk 1]
```diff
@@ -193,6 +199,9 @@ public static List<Column> toFullSchemasForHudiTable(Schema hudiSchema) {
         List<Schema.Field> allHudiColumns = hudiSchema.getFields();
         List<Column> fullSchema = Lists.newArrayList();
         for (Schema.Field fieldSchema : allHudiColumns) {
+            if (HUDI_INTERNAL_FIELD_NAMES.contains(fieldSchema.name())) {
```
Comment (DorianZheng) [inline fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java:202]: These can't be hide by default, you can add a property in create catalog statement to let user decide whether to hide it or not.

Comment (miomiocat) [reply fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java:202]: Agree.
Better to keep it and these fields are valuable when debugging

Comment (stephen-shelby) [reply fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java:202]: agree

**Comment by dirtysalt** [comment]
Since I don't have much time to follow this PR, i'd better to close it for now.

**Pull request closed without merging**